src/lib/kwlib.sh

```diff
@@ -291,6 +291,33 @@ function is_a_patch()
   return 0
 }
 
+function is_a_cover_letter()
+{
+  local -r FILE_PATH="$1"
+  local file_content
+
+  if [[ ! -f "$FILE_PATH" ]]; then
+    return 1
+  fi
+
+  file_content=$(< "$FILE_PATH")
+
+  local -ar COVER_LETTER_EXPECTED_STRINGS=(
+    'Subject:'
+    'To:'
+    'From:'
+    'Date:'
+  )
```

Comment:
Are you sure those strings are enough to determine whether it is a cover letter? Those are strings available in all patches generated by git. I think you need some specific heuristics, such as the patch number (it is usually 0) or another thing.
@davidbtadokoro do you know if there is a clear pattern for the cover letter? I'm asking you because you worked a lot in the lore part.